src/pyspextool/telluric/vega.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+import matplotlib.pyplot as pl
+import scipy
+import os
+
+from pyspextool.io.check import check_parameter
+from pyspextool.utils.arrays import find_index
+from pyspextool.fit.fit_peak1d import fit_peak1d
+from pyspextool.telluric import config as telluric
+from pyspextool.plot.limits import get_spec_range
+
+def vega_xcorrelate(object_wavelength, object_flux, vega_wavelength,
+                    vega_flux, vega_continuum, minimum_wavelength=None,
+                    maximum_wavelength=None, resolving_power=None,
+                    qa_show=False, qa_show_plotsize=(6,10), qa_fileinfo=None):
```

Comment:
consider doing the typehinting in the function definition.

Reply:
Done.